Create v1.1.3 release notes (#25)
harvester/release-notes: v1.1.3.md

# Harvester v1.1.3 Release Note

> This release follows Harvester v1.1.2, the documentation is available at https://docs.harvesterhci.io/v1.1.

Reviewer: ```suggestion
This release follows Harvester v1.1.2. The documentation is available at https://docs.harvesterhci.io/v1.1.
```

# Downloads

## ISO

:cd: [harvester-v1.1.3-amd64.iso](https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-amd64.iso)

## Additional Artifacts

:file_folder: https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-vmlinuz-amd64
:file_folder: https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-initrd-amd64
:file_folder: https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-rootfs-amd64.squashfs
:file_folder: https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-amd64.raw.zst
:white_check_mark: https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-amd64.sha512
:memo: https://releases.rancher.com/harvester/v1.2.1/version.yaml

# Install

Harvester can be installed with the [ISO](https://docs.harvesterhci.io/v1.1/install/iso-install/) image or via [PXE](https://docs.harvesterhci.io/v1.1/install/pxe-boot-install/) boot.

# Upgrade

> [Upgrade from v1.1.1/v1.1.2 to v1.1.3](https://docs.harvesterhci.io/v1.2/upgrade/index)

Reviewer: ```suggestion
You can [upgrade from v1.1.1 and v1.1.2 to v1.1.3](https://docs.harvesterhci.io/v1.2/upgrade/index).
```

# Bug Fixes

> - Project member can't view the created volume on the UI. See [#3874](https://github.com/harvester/harvester/issues/3874).
> - When Harvester deletes an uploading image, the corresponding temp file is not deleted. See [#4155](https://github.com/harvester/harvester/issues/4155). [Documentation](https://docs.harvesterhci.io/v1.1/upload-image#known-issues)

Reviewer: ```suggestion
### General Fixes

- Users that create a namespace and then create volumes in the same namespace are unable to view the new volumes on the Harvester UI. Previously, only the administrator was able to view the new volume data. See [issue #3874](https://github.com/harvester/harvester/issues/3874).

- When Harvester deletes an image file that is being uploaded, the corresponding temp file is also deleted. See [issue #4155](https://github.com/harvester/harvester/issues/4155) and the [documentation](https://docs.harvesterhci.io/v1.1/upload-image#known-issues).
```

> Upgrade:
> 
> - No PCI device displayed after upgrading to master and enabled the pcidevices-controller addons. See [#3994](https://github.com/harvester/harvester/issues/3994).
> - Fail to pack upgrade log: no space left on device. See [#4340](https://github.com/harvester/harvester/issues/4340). [Documentation](https://docs.harvesterhci.io/v1.1/upgrade/troubleshooting#download-upgrade-logs)

Reviewer: ```suggestion
### Upgrade Fixes

- The Harvester UI displays the available PCI devices after Harvester is upgraded and the pcidevices-controller addon is enabled. See [issue #3994](https://github.com/harvester/harvester/issues/3994).

- Upgrade log files are packaged correctly when users attempt to download the archive on the Harvester UI. The contents of the log files display as expected and the size of the archive is consistent each time the archive is downloaded. See [issue #4340](https://github.com/harvester/harvester/issues/4340) and the [documentation](https://docs.harvesterhci.io/v1.1/upgrade/troubleshooting#download-upgrade-logs).
```

> Installation:
> 
> - The installer occasionally doesn't run . See [#4767](https://github.com/harvester/harvester/issues/4767).
> - The Harvester ISO boot extremly slow on Dell XR4000. See [#4597](https://github.com/harvester/harvester/issues/4597).
> - Harvester OS stuck on boot with BOSS-S1 hardware RAID. See [#4334](https://github.com/harvester/harvester/issues/4334).

Reviewer: ```suggestion
### Installation Fixes

- The installer runs as expected when a network boot is performed using iPXE. See [issue #4767](https://github.com/harvester/harvester/issues/4767).

- The ISO boots at a normal speed and displays the installer on Dell PowerEdge XR4000 servers. See [issue #4597](https://github.com/harvester/harvester/issues/4597).

- The Harvester OS boots immediately when running on nodes with a Dell EMC BOSS-S1 card. See [issue #4334](https://github.com/harvester/harvester/issues/4334).
```

> # Dependency Component Versions

Reviewer: ```suggestion
## Component Versions
```

| Component        | Version                                                                         |
|------------------|---------------------------------------------------------------------------------|
| Longhorn         | [v1.4.3](https://github.com/longhorn/longhorn/releases/tag/v1.4.3)              |
| KubeVirt         | [v0.54.0](https://github.com/kubevirt/kubevirt/releases/tag/v0.54.0)            |
| Embedded Rancher | [v2.7.5](https://github.com/rancher/rancher/releases/tag/v2.6.10)               |
| RKE2             | [v1.25.9+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.25.9%2Brke2r1) |

> # Contributors
> 
> Thanks to all the contributors that make the Harvester v1.1.3 release possible!

Reviewer: ```suggestion
## Contributors
```

# Harvester v1.1.3 Release Note

This release follows Harvester v1.1.2. The documentation is available at https://docs.harvesterhci.io/v1.1.

## Downloads

### ISO

:cd: [harvester-v1.1.3-amd64.iso](https://releases.rancher.com/harvester/v1.1.3/harvester-v1.1.3-amd64.iso)

### Additional Artifacts

:file_folder: https://releases.rancher.com/harvester/v1.1.3/harvester-v1.1.3-vmlinuz-amd64
:file_folder: https://releases.rancher.com/harvester/v1.1.3/harvester-v1.1.3-initrd-amd64
:file_folder: https://releases.rancher.com/harvester/v1.1.3/harvester-v1.1.3-rootfs-amd64.squashfs
:white_check_mark: https://releases.rancher.com/harvester/v1.1.3/harvester-v1.1.3-amd64.sha512
:memo: https://releases.rancher.com/harvester/v1.1.3/version.yaml

## Installation

> Harvester can be installed using the [ISO image](https://docs.harvesterhci.io/v1.1/install/iso-install/), a [bootable USB drive](https://docs.harvesterhci.io/v1.2/install/usb-install), and [PXE boot](https://docs.harvesterhci.io/v1.1/install/pxe-boot-install/).

Reviewer 2: ```suggestion
Harvester can be installed using the [ISO image](https://docs.harvesterhci.io/v1.1/install/iso-install/), a [bootable USB drive](https://docs.harvesterhci.io/v1.1/install/usb-install), and [PXE boot](https://docs.harvesterhci.io/v1.1/install/pxe-boot-install/).
```

# Harvester v1.1.3 Release Note

This release follows Harvester v1.1.2. The documentation is available at https://docs.harvesterhci.io/v1.1.

## Downloads

### ISO

:cd: [harvester-v1.1.3-amd64.iso](https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-amd64.iso)

### Additional Artifacts

:file_folder: https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-vmlinuz-amd64
:file_folder: https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-initrd-amd64
:file_folder: https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-rootfs-amd64.squashfs
:file_folder: https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-amd64.raw.zst
:white_check_mark: https://releases.rancher.com/harvester/v1.2.1/harvester-v1.2.1-amd64.sha512

> :memo: https://releases.rancher.com/harvester/v1.2.1/version.yaml

Author: @markhillgit May you update the ISO file and Additonal Artifacts when ready? Thanks!

### Installation Fixes

- The installer runs as expected when a network boot is performed using iPXE. See [issue #4767](https://github.com/harvester/harvester/issues/4767).

- The ISO boots at a normal speed and displays the installer on Dell PowerEdge XR4000 servers. See [issue #4597](https://github.com/harvester/harvester/issues/4597).

- The Harvester OS boots immediately when running on nodes with a Dell EMC BOSS-S1 card. See [issue #4334](https://github.com/harvester/harvester/issues/4334).

## Component Versions

| Component        | Version                                                                         |
|------------------|---------------------------------------------------------------------------------|
| Longhorn         | [v1.4.3](https://github.com/longhorn/longhorn/releases/tag/v1.4.3)              |
| KubeVirt         | [v0.54.0](https://github.com/kubevirt/kubevirt/releases/tag/v0.54.0)            |
| Embedded Rancher | [v2.7.5](https://github.com/rancher/rancher/releases/tag/v2.6.10)               |

> | RKE2             | [v1.25.9+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.25.9%2Brke2r1) |

Author: @markhillgit May you update the Component Versions when ready? Thanks!
